catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/ResourceOperationsTest.java

```diff
@@ -0,0 +1,443 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package ddf.catalog.impl.operations;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyBoolean;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.io.Serializable;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import ddf.catalog.cache.solr.impl.ValidationQueryFactory;
+import ddf.catalog.content.impl.MockMemoryStorageProvider;
+import ddf.catalog.data.Result;
+import ddf.catalog.data.impl.MetacardImpl;
+import ddf.catalog.federation.FederationStrategy;
+import ddf.catalog.filter.proxy.adapter.GeotoolsFilterAdapterImpl;
+import ddf.catalog.filter.proxy.builder.GeotoolsFilterBuilder;
+import ddf.catalog.impl.FrameworkProperties;
+import ddf.catalog.impl.QueryResponsePostProcessor;
+import ddf.catalog.operation.QueryRequest;
+import ddf.catalog.operation.QueryResponse;
+import ddf.catalog.operation.ResourceRequest;
+import ddf.catalog.operation.ResourceResponse;
+import ddf.catalog.plugin.PostIngestPlugin;
+import ddf.catalog.plugin.PostResourcePlugin;
+import ddf.catalog.plugin.StopProcessingException;
+import ddf.catalog.resource.DataUsageLimitExceededException;
+import ddf.catalog.resource.Resource;
+import ddf.catalog.resource.ResourceNotFoundException;
+import ddf.catalog.resource.ResourceNotSupportedException;
+import ddf.catalog.resource.download.DownloadException;
+import ddf.catalog.resource.download.ReliableResourceDownloadManager;
+import ddf.catalog.resourceretriever.ResourceRetriever;
+import ddf.catalog.source.CatalogProvider;
+import ddf.catalog.util.impl.SourcePoller;
+import ddf.mime.MimeTypeResolver;
+import ddf.mime.MimeTypeToTransformerMapper;
+import ddf.mime.mapper.MimeTypeMapperImpl;
+
+public class ResourceOperationsTest {
```

Comment:
❗️ Is there a reason you created a whole new junit test class for this instead of just adding tests to the existing Spock test class?

Also, by creating this class with an identical name to the Spock test, you're stomping on one or the other at compile time (I could confirm which, but I believe it is the Groovy code which will stomp on this test); only one of the test classes will survive compilation and only one of the test classes will then actually be run.

I would advise moving and translating the tests in this class to the extant Spock test class; barring that, you'll need to rename one or the other of these to disambiguate them and prevent the collisions you've caused.